src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,32 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
```

Comment:
You should use POST for adding driver. Add `protected void doPost(HttpServletRequest req, HttpServletResponse resp)` to this controller.